crates/red_knot_python_semantic/resources/mdtest/attributes.md

```diff
@@ -213,11 +475,20 @@ reveal_type(True.numerator)  # revealed: Literal[1]
 reveal_type(False.real)  # revealed: Literal[0]
 ```
 
-## Bytes-literal attributes
+### Bytes-literal attributes
 
 All attribute access on literal `bytes` types is currently delegated to `buitins.bytes`:
 
 ```py
 reveal_type(b"foo".join)  # revealed: @Todo(instance attributes)
 reveal_type(b"foo".endswith)  # revealed: @Todo(instance attributes)
 ```
+
+## References
+
+Some of the tests in the *Class and instance variables* section draw inspiration from
+[pyright's documentation] on this topic.
+
+[pyright's documentation]: https://microsoft.github.io/pyright/#/type-concepts-advanced?id=class-and-instance-variables
+[typing documentation on `classvar`]: https://typing.readthedocs.io/en/latest/spec/class-compat.html#classvar
```

Comment:
nit: this actually links to the spec rather than docs that are meant to be user-facing. (I think that's the correct page for us to link to here but "typing documentation on `classvar`" is maybe a slightly confusing way to describe the page?)

Reply:
Ahrg, I had "typing spec on …" first, then changed it to documentation after I saw the title of the page. Changing back. The `classvar` thing is done by mdformat. I actually wrote `ClassVar`, but that get's lowercased.